cosmwasm/submit-proposal.js

```diff
@@ -92,7 +92,10 @@ const storeCode = async (client, wallet, config, options) => {
     const proposalId = await callSubmitProposal(client, wallet, config, options, proposal);
 
     contractBaseConfig.storeCodeProposalId = proposalId;
-    contractBaseConfig.storeCodeProposalCodeHash = createHash('sha256').update(readWasmFile(options)).digest().toString('hex');
+    contractBaseConfig.storeCodeProposalCodeHash = createHash('sha256')
+        .update(readFileSync(options.wasmResolvedPath))
```

Comment:
```suggestion
        .update(readContractCode(options))
```
abstract away the details

Reply:
done